src/scss/grid.scss

```diff
@@ -0,0 +1,235 @@
+/**
+ * Profound Grid
```

Comment:
I'd vote to put this in a vendor or lib folder within scss to more clearly indicate that it's 3rd-party code